forum/Sources/Tips.php

```diff
@@ -3,12 +3,16 @@
 if (!defined('SMF'))
 	die('Hacking attempt...');
 
+include_once("Paginate.php");
+
 function Tips()
 {
-	global $context, $smcFunc, $user_info, $txt;
+	global $context, $smcFunc, $user_info, $txt, $modSettings;
 
 	$context['page_title'] = $txt['tip_list_title'];
+	$modSettings['disableQueryCheck'] = true;

```

Comment:
What does this do exactly?

Reply:
SMF's query function won't let you use subqueries unless you set that value.